python/docs/source/development/ps_contributing.rst

```diff
@@ -1,192 +0,0 @@
-==================
-Contributing Guide
-==================
-
-.. contents:: Table of contents:
-   :depth: 1
-   :local:
-
-Types of Contributions
-======================
-
-The largest amount of work consists simply of implementing the pandas API using Spark's built-in functions, which is usually straightforward. But there are many different forms of contributions in addition to writing code:
-
-1. Use the project and provide feedback, by creating new tickets or commenting on existing relevant tickets.
-
-2. Review existing pull requests.
-
-3. Improve the project's documentation.
-
-4. Write blog posts or tutorial articles evangelizing pandas API on Spark and help new users learn pandas API on Spark.
-
-5. Give a talk about pandas API on Spark at your local meetup or a conference.
-
-
-Step-by-step Guide For Code Contributions
```

Comment:
This section was removed too:
- Design principles (1.) are merged to "Preparing to Contribute Code Changes"
- 2., 3., 4., 5., 6. and 7. are duplicates with https://spark.apache.org/docs/latest/api/python/development/contributing.html#contributing-to-pyspark and https://spark.apache.org/contributing.html